### PR TITLE
Spring boot 2.1.0

### PR DESCRIPTION
**Pull request by DanielFran**

### CODE REVIEW

File: jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterLoggingMetricsExportConfiguration.java [hunk 1]
```diff
@@ -1,80 +0,0 @@
-package io.github.jhipster.config.metric;
```
Comment (PierreBesson) [inline jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterLoggingMetricsExportConfiguration.java]: Why is this class removed !?

File: jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterMetricsEndpointConfiguration.java [hunk 2]
```diff
@@ -1,26 +0,0 @@
-package io.github.jhipster.config.metric;
```
Comment (PierreBesson) [inline jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterMetricsEndpointConfiguration.java]: Same for this one, this is required for micrometer.

File: jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterLoggingMetricsExportConfiguration.java (hunk 1 above)
Comment (DanielFran) [reply jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterLoggingMetricsExportConfiguration.java]: Probably a bad merged from master to the sb branch

File: jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterMetricsEndpointConfiguration.java (hunk 2 above)
Comment (DanielFran) [reply jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterMetricsEndpointConfiguration.java]: Same

**Comment by pascalgrimaud** [comment]
Important : as soon as this PR is merged, we need to change the value from `spring-boot_2.1.0` to `master`, in the main `generator-jhipster` project:
- https://github.com/jhipster/generator-jhipster/blob/master/.travis.yml#L44-L49
- https://github.com/jhipster/generator-jhipster/blob/master/azure-pipelines.yml#L28-L33

File: .travis.yml [hunk 3]
```diff
@@ -49,7 +49,7 @@ env:
     - JHI_LIB_BRANCH=master
     # if JHI_GEN_BRANCH value is release, use the release from NPM
     - JHI_GEN_REPO=https://github.com/jhipster/generator-jhipster.git
-    - JHI_GEN_BRANCH=master
+    - JHI_GEN_BRANCH=spring-boot_2.1.0
```
Comment (pascalgrimaud) [inline .travis.yml]: as spring-boot_2.1.0 branch in generator-jhipster project has been merged, we can switch here to master

File: jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterLoggingMetricsExportConfiguration.java (hunk 1 above)
Comment (DanielFran) [reply jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterLoggingMetricsExportConfiguration.java]: I added back files

File: jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterMetricsEndpointConfiguration.java (hunk 2 above)
Comment (DanielFran) [reply jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterMetricsEndpointConfiguration.java]: I added back files

File: .travis.yml (hunk 3 above)
Comment (DanielFran) [reply .travis.yml]: switched back to master generator branch

**Pull request merged**